src/components/course/data/utils.jsx

```diff
@@ -244,7 +244,11 @@ export function findHighestLevelSeatSku(seats) {
   }
   const courseModes = seats.map(seat => seat.type);
   const courseMode = getBestCourseMode(courseModes);
-  return seats.find(seat => seat.type === courseMode)?.sku;
+  if (courseMode) {
+    return seats.find(seat => seat.type === courseMode)?.sku;
+  }
+  // Return first seat if the type isn't included in the list
+  return seats?.[0];
```

Comment:
Should Exec Ed courses have a new type value, or will they use the existing type values found in [COURSE_MODES_MAP](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/c4af2a8b08c52cd0aee639a67a9d82c73e344b2e/src/components/course/data/constants.js#L35)?